lib/chef/resource/osx_profile.rb

```diff
@@ -30,7 +30,7 @@ class OsxProfile < Chef::Resource
       provides :osx_profile
       provides :osx_config_profile
 
-      description "Use the **osx_profile** resource to manage configuration profiles (`.mobileconfig` files) on the macOS platform. The **osx_profile** resource installs profiles by using the uuidgen library to generate a unique `ProfileUUID`, and then using the `profiles` command to install the profile on the system."
+      description "Use the **osx_profile** resource to manage configuration profiles (`.mobileconfig` files) on the macOS platform. The **osx_profile** resource installs profiles by using the uuidgen library to generate a unique `ProfileUUID`, and then using the `profiles` command to install the profile on the system. Local profile installation is no longer supported by Apple on macOS 10.16+"
```

Comment:
```suggestion
      description "Use the **osx_profile** resource to manage configuration profiles (`.mobileconfig` files) on the macOS platform. The **osx_profile** resource installs profiles by using the uuidgen library to generate a unique `ProfileUUID`, and then using the `profiles` command to install the profile on the system. Warning: Local profile installation is no longer supported by Apple on macOS Big Sur or above."
```

First of all, great idea! Not mentioning this in the docs is a pretty big oversight on our part. I had few suggestions though:

* Our docs are partially generated from these fields in the resources, and if you preface a string with `Warning: ` it'll get turned into a more noticeable box like the one on this page: https://docs.chef.io/resources/chef_gem/
* For consistency the language should probably match that of the similar error message: https://github.com/chef/chef/blob/aca7925a8ad7612d8bd79475858c8008cb4d6f89/lib/chef/resource/osx_profile.rb#L162